### PR TITLE
Implementing CategoryViewController Skeleton

### PR DESCRIPTION
**Pull request by AnnaBanani**
Implementing CategoryViewController Skeleton

### CODE REVIEW

File: TheMET/TheMET/ViewControllers/CategoryViewController.swift [hunk 1]
```diff
@@ -8,18 +8,140 @@
...
+        case failed
+    }
+    
+    var contentStatus: Status = .loading {
+        didSet {
+            self.updateContent()
+        }
+    }
+    
+    override func awakeFromNib() {
```
Comment (Ssimboss) [inline TheMET/TheMET/ViewControllers/CategoryViewController.swift:34]: This method override doesn't do anything

File: TheMET/TheMET/ViewControllers/CategoryViewController.swift [hunk 2]
```diff
@@ -8,18 +8,140 @@
...
+
+    private let favoriteService = FavoritesService.standart
+    
+    private let loadingCategoryView = LoadingPlaceholderView.constructView(configuration: .categoryArtworksLoading)
+    private let failedCategoryView = LoadingPlaceholderView.constructView(configuration: .categoryFailed)
+    
+    private let categoryTableView: UITableView = UITableView(frame: .zero, style: .plain)
+    
+    enum Status {
+        case loaded
```
Comment (Ssimboss) [inline TheMET/TheMET/ViewControllers/CategoryViewController.swift:23]: `loaded` is supposed to have an associated array of `Art`-objects

File: TheMET/TheMET/ViewControllers/CategoryViewController.swift [hunk 3]
```diff
@@ -8,18 +8,140 @@
 import Foundation
 import UIKit
 
-class CategoryViewController: UIViewController {
+class CategoryViewController: UIViewController, UITableViewDelegate, UITableViewDataSource  {
     
-    @IBOutlet
-    private var cancelButon: UIButton!
+    var titleCategoria: String = String()
```
Comment (Ssimboss) [inline TheMET/TheMET/ViewControllers/CategoryViewController.swift:13]: What is this property for?

**Review by Ssimboss — CHANGES_REQUESTED** [review]

File: TheMET/TheMET/ViewControllers/CategoryViewController.swift (hunk 1 above)
Comment (AnnaBanani) [reply TheMET/TheMET/ViewControllers/CategoryViewController.swift:34]: +

**Pull request merged**